test/checkpoint/test_checkpoint.c

```diff
@@ -217,6 +273,21 @@ main(int argc, char *argv[])
               g.cookies[i].uri, sizeof(g.cookies[i].uri), "%s%04d", URI_BASE, g.cookies[i].id));
         }
 
+        /*
+         * Setup thread data. There are N worker threads, a checkpoint thread and possibly a clock
+         * thread. The workers are ID 0 to N-1, checkpoint thread as N, and the clock thread as N +
+         * 1.
+         */
+        if ((g.td = calloc((size_t)(g.nworkers + 2), sizeof(THREAD_DATA))) == NULL) {
+            (void)log_print_err("No memory", ENOMEM, 1);
+            break;
+        }
+        for (i = 0; i < g.nworkers; ++i) {
+            init_thread_data(&g.td[i], i);
```

Comment:
Formatting - unneeded braces for single line loop.

Reply:
Fixed